lib/ts/recipe/passwordless/types.ts

```diff
@@ -59,12 +59,13 @@ export type TypeInput = (
                   // Unlikely, but someone could display this (or a derived thing) to identify the device
                   preAuthSessionId: string;
               },
+              tenantId: string,
```

Comment:
remove the deprecated createAndSendCustomTextMessage and other functions from other recipes

Reply:
Done